integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -504,18 +504,13 @@ public static V1Pod getPod(String namespace, String labelSelector, String podNam
    * @param serviceName name of the service to check for
    * @param label the key value pair with which the service is decorated with
    * @param namespace the namespace in which to check for the service
-   * @return true if the service is found otherwise false
+   * @return true if the service is found, otherwise false.
    * @throws ApiException when there is error in querying the cluster
    */
   public static boolean doesServiceExist(
       String serviceName, Map<String, String> label, String namespace)
       throws ApiException {
-    boolean exist = false;
-    V1Service service = getService(serviceName, label, namespace);
-    if (service != null) {
-      exist = true;
-    }
-    return exist;
+    return getService(serviceName, label, namespace) != null ? true : false;
```

Comment:
The ternary operator is redundant. You can just return the result of the null check:
```
return getService(serviceName, label, namespace) != null;
```

Reply:
Changed in [3bfacc5](https://github.com/oracle/weblogic-kubernetes-operator/pull/3060/commits/3bfacc537c300681c87acf04c7797a7cf0355b3e).